api/errors.go

```diff
@@ -93,3 +93,8 @@ type Error string
 
 // Error returns the error message.
 func (e Error) Error() string { return string(e) }
+
+// Webhook errors
+const (
+	ErrWebhookAlreadyExists = Error("A webhooks with the same name already exists")
```

Comment:
I think the error message is invalid. We do not associate any name to a wehbook.